netlify.toml

```diff
@@ -2,3 +2,6 @@
   from = "/*"
   to = "/"
   status = 200
+[build]
+  node_version = "18.18.0"
```

Comment:
is this needed with the `.nvmrc? My understanding is that one or the other is needed but not both, and I'm afraid of seeing them different in the future :-)